packages/vm/tests/api/runBlock.js

```diff
@@ -146,6 +146,74 @@ tape('should run valid block', async (t) => {
   t.end()
 })
 
+// this test actually checks if the DAO fork works. This is not checked in ethereum/tests
+tape('should transfer balance from DAO children to the Refund DAO account in the DAO fork', async (t) => {
+// here: get the default fork list of mainnet and only edit the DAO fork block (thus copy the rest of the "default" hardfork settings)
+  const defaultCommon = new Common('mainnet', "dao")
+  // retrieve the hard forks list from defaultCommon...
+  let forks = defaultCommon.hardforks()
+  let editedForks = []
+  // explicitly edit the "dao" block number:
+  for (let fork of forks) {
+    if (fork.name == "dao") {
+      editedForks.push({
+        name: "dao",
+        forkHash: fork.forkHash,
+        block: 1
+      })
+    } else {
+      editedForks.push(fork)
+    }
+  }
+  let common = Common.forCustomChain('mainnet', {
+    hardforks: editedForks
+  }, 'homestead') // we should be on the "homestead" fork
```

Comment:
Ah, just seeing that this code is repeated here again. Can you then put it into `tests/util.js` as its own function? Then I think it is also viable to leave "as is".

Reply:
Ah yes this is a good solution.